src/app/src/advancedsetupdialog.ui

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>advancedsetupdialog</class>
```

Comment:
you can probably ignore my other comments, it's probably not working because this name is all lowercase but you're using the camel case name. You probably need to change the name in QT Creator to make it match the case in the cpp/h files